### PR TITLE
Set vscan_fileop_profile for na_ontap_cifs

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Set vscan_fileop_profile for na_ontap_cifs

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME
- na_ontap_cifs.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: test/units/modules/storage/netapp/test_na_ontap_cifs.py [hunk 1]
```diff
@@ -75,8 +75,11 @@ def build_cifs_info():
         data = {'num-records': 1, 'attributes-list': {'cifs-share': {
             'share-name': 'test',
             'path': '/test',
-            'share-properties': {'cifs-share-properties': 'browsable'},
-            'symlink-properties': {'cifs-share-symlink-properties': 'enable'},
+            'vscan-fileop-profile': 'standard',
+            'share-properties': {'cifs-share-properties': 'browsable',
+                                 'cifs-share-properties': 'oplocks'},
```
Comment (lonico) [inline test/units/modules/storage/netapp/test_na_ontap_cifs.py]: we can't have the same key

File: test/units/modules/storage/netapp/test_na_ontap_cifs.py [hunk 2]
```diff
@@ -75,8 +75,11 @@ def build_cifs_info():
...
+            'symlink-properties': {'cifs-share-symlink-properties': 'enable',
+                                   'cifs-share-symlink-properties': 'read_only'},
```
Comment (lonico) [inline test/units/modules/storage/netapp/test_na_ontap_cifs.py]: same here

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — APPROVED** [review]
lgtm, tests updated and docs updated.

**Review by thedoubl3j — APPROVED** [review]

**Pull request merged**